.docs/contributing-book/src/documentation/project-quality/project-structure/external.md

```diff
@@ -0,0 +1,37 @@
+# External Project
+
+These are projects that expose an interface, e.g., the `ABI` of your contract(s), that can be imported from other projects.
+
+Here is an example structure for such an application that consists of two separate projects. In this example, `my_library` contains the `ABI` of the contract. `my_contract` depends on `my_library` and contains the implementation of the `ABI`. This structure allows `my_library` to be imported from outside `my_application`.
```

Comment:
```suggestion
The following structure separates the interface of the contract into its own library project so that it can be imported alongside projects in the `my_application` directory and outside of it.
```